docs/user/quick-start/installation/index.md

```diff
@@ -5,57 +5,75 @@ summary: Installation Guide for multiple operating systems
 
 # Preparing to install
 
-The first step to installing Solus is acquiring the correct media. The Solus Project provides ISO images, which contain the contents of Solus for installing.
+The first step to installing Solus is getting the correct media. The Solus Project provides ISO images, which contain the contents of Solus for installing.
 
----
-
-# Getting the ISO
+## Getting the ISO
 
-You can download a Solus ISO by going to our [Download page](https://getsol.us/download).
+Download a Solus ISO by going to our [Download page](https://getsol.us/download) and picking an edition.
 
----
+## Verifying the ISO
 
-# Verifying the ISO
+### Linux and macOS
 
-## Linux and Mac
+Download the SHA256SUMS files for your chosen ISO edition: _File_, _Signed File_ and _Public Key_ from the links next to the _Download_ buttons on our [Download page](https://getsol.us/download).
 
-Download the relevant Solus ISO SHA256SUMS: File, Signed File and Public Key from the [Download page](https://getsol.us/download).
+**Note:** You should download all these files to the same directory.
 
-**Note:** It is advisable that the files are all placed in the same folder.
+#### Verifying the SHA256SUMS file signature
 
-### Import the Public Key
+Import the Solus public key:
 
 ```bash
-gpg --import <Public Key>.gpg
+cd ~/path/to/download/directory
+gpg --import solus-releng-pub.gpg
 ```
 
-### Verify SHA256SUMS file signature
+Verify the signed checksum file:
 
 ```bash
-gpg --verify <Signed File>.sign <File>.sha256sum
+gpg --verify Solus-4.4-Budgie.iso.sha256sum.sign Solus-4.4-Budgie.iso.sha256sum
 ```
 
-### Verify ISO
+#### Verifying the ISO checksum
+
+Check that the computed hash of the downloaded ISO file matches the hash supplied by Solus:
 
 ```bash
 sha256sum -c Solus-4.4-Budgie.iso.sha256sum | grep OK
 ```
 
----
+### Windows
+
+#### Verifying signed SHA256SUM file
+
+On Windows, first install [gpg4win](https://www.gpg4win.org/).
+
+Now you can run the same `gpg` commands from the the _Verify SHA256SUMS file signature_ section above.
```

Comment:
Do we want to link to the section above? Saves the reader from scrolling to find it. I can never remember the syntax for a link like that, though.

```markdown
[Verify SHA256SUMS file signature](#Verifying-the-SHA256SUMS-file-signature)
```

Maybe? You'd have to play around with it.

Reply:
Maybe there's a syntax for links within the same file. Too lazy to find out, use absolute path instead